setup.py

```diff
@@ -89,6 +89,16 @@
         except pkg_resources.DistributionNotFound:
             version = "n/a"
         print(f"    {dist}: {version}")
+    version = "n/a"
+    if shutil.which("rustc") is not None:
+        try:
+            # If for any reason `rustc --version` fails, silently ignore it
+            rustc_output = subprocess.run(["rustc", "--version"], capture_output=True, timeout=0.5, encoding="utf8").stdout
+            version = rustc_output.rpartition(" ")[-1]
```

Comment:
```suggestion
            version = subprocess.run(["rustc", "--version"], capture_output=True, timeout=0.5, encoding="utf8").stdout.strip()
```

I think this is fine, full detail is more appropriate here.

Reply:
How about `.strip().remove_prefix("rustc ")`?

Reply:
That'd be fine, except `removeprefix` looks like it wasn't added until 3.9 and we support 3.6 😬

Reply:
`re.sub("^rustc ", "", rustc_output.strip())`?